app/models/role.rb

```diff
@@ -2,15 +2,28 @@
 require "menu"
 
 class Role < ActiveRecord::Base
+  include ApplicationHelper
```

Comment:
Include `ApplicationHelper` in model file is not a good idea, there should be some other ways to do it.